addons/default/visiosoft/base-theme/resources/views/addons/anomaly/users-module/register.twig

```diff
@@ -11,7 +11,7 @@
         <div class="row justify-content-center bg-light rounded border px-3 register-area">
             <div class="col-12 col-md-6 text-left p-3 m-0">
 
-                <div class="personal-register">
+                <div class="personal-register {{ setting_value('visiosoft.module.store::hide_personal_register') ? 'd-none' }}">
```

Comment:
remove store setting

Reply:
The corporate registration screen comes from the store-module, where should I put it @diashalabi

Reply:
you can use css for example and put it in the store module